### DISCUSSION TITLE
[New Bugs]

### DISCUSSION THREAD

**Discussion by zhnjsacdjkn** [Phobos-developers/Phobos#604] [Ideas and feedback]
In the recent b27 version, warheads with 'conventional=yes' code (splashing) and crit codes generate IE when attacking water units (crit will cause IE, no crit will be fine, just targeting water units, land units are fine ), hope to fix

https://user-images.githubusercontent.com/98029842/164159312-ffc0eb0f-ef17-4048-a3bb-63fa84a6719b.mp4

![ie](https://user-images.githubusercontent.com/98029842/164159491-709cd6cd-ad82-4f6e-a103-b2f2c04700bf.png)

**Comment by Starkku** [comment]
Show related INI code ~~and list any other patches you might be using (DP/Kratos etc).~~

EDIT: I noticed you are using DP/Kratos. This issue is almost certainly an incompatibility with the said patch given that nobody else is experiencing this problem. Phobos is not designed to function alongside with any other patch or engine extension than Ares so combined use of any other extensions is at your own discretion. Any bugs and problems encountered while doing so should first be tested *without* the other extensions to rule out incompatibility issues before reporting them to Phobos developers.

Also bug reports go to Issues, not Discussions.

**Comment by AlliedG** [comment]
I did not encounter an internal error when destroying an amphibious transport on water with the latest development nightly build of Phobos.

I only use Ares + Phobos

Relevant code I used:

```
Conventional=yes
Crit.Chance=100%
Crit.ExtraDamage=15
Crit.Warhead=AP
Crit.Affects=units,buildings
```

**Reply by xindun110** [reply]
带有RadLevel=辐射的武器，遇到水则崩溃！
